Firmware/AVR/AVR_DRIVER/lib/encoder/encoder.h

```diff
@@ -0,0 +1,53 @@
+
+/**
+ * @file encoder.h
+ * @author Tsugumi Murata (tmurata293)
+ * @date 18 Feb 2021
+ * @brief library for encoder 
+ *
+ */
+
+#ifndef _ENCODER_H_
+#define _ENCODER_H_
+
+
+#ifdef __cplusplus
+extern "C"{
+#endif 
+
+#include <stdio.h>
+#include "../../include/pinConfig.h"
+
+#define ENCODER_SIG_MASK (_BV(ENCODER_SIG_A) | _BV(ENCODER_SIG_B) )
+
+#define ENCODER16_FIRST_8BIT             0xFF00
```

Comment:
MINOR: good practice to (0xFF00)

You never know if you would forget () when use macros

Reply:
added